modules/identity-provider-oidc-about.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-oidc-identity-provider.adoc
+
+[id="identity-provider-oidc-about_{context}"]
+= About OpenID Connect authentication
+
+ifdef::openshift-origin[]
+You can link:https://www.keycloak.org/docs/latest/server_admin/index.html#openshift[configure a Keycloak] server as an OpenID Connect identity provider for {product-title}.
+endif::[]
```

Comment:
Can you supply a preview link to the OKD docs to view this change?

Reply:
Added to comment 0: http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-2973-okd/authentication/identity_providers/configuring-oidc-identity-provider.html#identity-provider-oidc-about_configuring-oidc-identity-provider